webpack.npm-library-config.js

```diff
@@ -0,0 +1,73 @@
+const path = require('path')
+const CopyPlugin = require('copy-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const stringReplacement = require('./build-support/string-replacement')
+const {
+  entry,
+  dest,
+  assetDest,
+  assets,
+  outputFileName
+} = require('./build-support/build-opts')
+
+// TODO: This is just a placeholder for a webpack config for publishing the CFM
+// as a set of NPM modules. ( TBD obviously)
```

Comment:
A minor point, but my inclination would be to leave this out of this PR. You can check it in on a branch if you think it would be useful, but you might end up not even needing a separate webpack-config to support the npm library case.